README.md

```diff
@@ -10,44 +10,68 @@
 [![Coverage Status](https://codecov.io/gh/pingcap/tidb/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/tidb)
 [![GoDoc](https://img.shields.io/badge/Godoc-reference-blue.svg)](https://godoc.org/github.com/pingcap/tidb)
 
-- [**Slack Channel**](https://pingcap.com/tidbslack/)
-- **Twitter**: [@PingCAP](https://twitter.com/PingCAP)
-- [**Reddit**](https://www.reddit.com/r/TiDB/)
-- **Mailing list**: [Google Group](https://groups.google.com/forum/#!forum/tidb-user)
-- [**Blog**](https://www.pingcap.com/blog/)
-- [**For support, please contact PingCAP**](http://bit.ly/contact_us_via_github)
+<!-- vim-markdown-toc GFM -->
+
+* [What is TiDB?](#what-is-tidb)
+* [Documentation](#documentation)
+* [Quick start](#quick-start)
+* [Deployment](#deployment)
+* [Data Migration](#data-migration)
+* [Getting Help](#getting-help)
+* [Adopters](#adopters)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+* [Acknowledgments](#acknowledgments)
+
+<!-- vim-markdown-toc -->
 
 ## What is TiDB?
 
 TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability.
 
-- __Horizontal Scalability__
+For a detailed introduction to TiDB architecture, see [TiDB Architecture](https://pingcap.com/docs/dev/architecture/).
```

Comment:
I recommend that we keep the architecture diagram here so users won't have to jump through different links to find out.